### PR TITLE
[ALLUXIO-2060] [DOCFIX] Improve flink docs.

### PR DESCRIPTION
**Pull request by ifcharming**
Addresses: https://alluxio.atlassian.net/browse/ALLUXIO-2060

### CODE REVIEW

**Comment by ifcharming** [comment]
@apc999 @jsimsa

File: docs/en/Running-Flink-on-Alluxio.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ available on all cluster nodes as well). For example like this:
 
 {% include Running-Flink-on-Alluxio/hadoop-classpath.md %}
 
+#### Translate additional Alluxio site properties to Flink
+
+In additional, if there are any customized Alluxio site properties in
```
Comment (jsimsa) [inline docs/en/Running-Flink-on-Alluxio.md]: In addition

Comment (jsimsa) [inline docs/en/Running-Flink-on-Alluxio.md]: if there are any properties specified in `conf/...`, translate those to ...

File: docs/en/Running-Flink-on-Alluxio.md [hunk 2]
```diff
@@ -65,6 +65,12 @@ available on all cluster nodes as well). For example like this:
...
+`conf/alluxio-site.properties`, please translate those to `env.java.opts` in
+`{FLINK_HOME}/conf/flink-conf.yaml` for Flink to pick up Alluxio configurations.
```
Comment (jsimsa) [inline docs/en/Running-Flink-on-Alluxio.md]: s/configurations/configuration

**Comment by jsimsa** [comment]
@ifcharming thanks, left some minor comments

File: docs/en/Running-Flink-on-Alluxio.md (hunk 1 above)
Comment (ifcharming) [inline docs/en/Running-Flink-on-Alluxio.md]: Ooops, updated, thanks.

File: docs/en/Running-Flink-on-Alluxio.md (hunk 2 above)
Comment (ifcharming) [inline docs/en/Running-Flink-on-Alluxio.md]: done

**Comment by ifcharming** [comment]
Thanks @jsimsa , I've updated the PR. PTAL

**Comment by jsimsa** [comment]
LGTM

**Comment by jsimsa** [comment]
Jenkins, test this please

**Comment by jsimsa** [comment]
Jenkins, test this please

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**